### DISCUSSION TITLE
Formatting a Tabe using Grafana and Prometheus/WindowsExporter

### DISCUSSION THREAD

**Discussion by ghost** [grafana/grafana#63428] [Dashboards, Panels & Visualizations]
I want to create a table in Grafana with WindowsExporter/Prometheus by displaying Exchange-specific values. However, these values always return a "name" (the command) and a "name" (the variable). Accordingly, the next command is displayed subordinate to me and not in a row. Does anyone know how to fix this?

Not solving Problems: hide with "transform", hide with "override".
![help](https://user-images.githubusercontent.com/125640633/220033231-e31f537b-d66c-4632-92b9-2bd08bb7beba.PNG)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
